ts/navigation/NavigationService.ts

```diff
@@ -39,13 +33,6 @@ const setTopLevelNavigator = (
   navigatorRef: NavigationContainerComponent | null | undefined
 ) => {
   // TODO: remove when the bug is confirmed as solved
-  instabugLog(
```

Comment:
I guess we can remove also the `TODO` line above

Reply:
Sure, removed here [8bd6bef](https://github.com/pagopa/io-app/pull/3867/commits/8bd6befc2441d87de8a44e3b2d0873ea9635f5c1)